### PR TITLE
Feat/polls

### PR DESCRIPTION
**Pull request by Eccleria**
Ready for 5.0.0 pre release

### CODE REVIEW

File: src/commands/spotify.js
Comment (Titch88) [inline src/commands/spotify.js:1]: this entire file is currently unused in prod

File: src/commons.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return this.prod;
+  }
+  getShared() {
+    return this.shared;
+  }
+  fetchGuildId(guildId) {
+    return this.list.find((obj) => guildId === obj.guildId);
+  }
+  getBoth() {
+    return [this.test, this.prod];
```
Comment (Titch88) [inline src/commons.js:26]: more like return this.list

**Review by Titch88 — APPROVED** [review]

File: src/commons.js (hunk 1 above)
Comment (Eccleria) [reply src/commons.js:26]: Fixed in #181

File: src/commands/spotify.js
Comment (Eccleria) [reply src/commands/spotify.js:1]: Will be examined in #182

**Pull request merged**